mk/spksrc.cross-cmake-env.mk

```diff
@@ -6,11 +6,8 @@ CMAKE_ARGS += -DCMAKE_SYSTEM_NAME=Linux
 CMAKE_ARGS += -DCMAKE_BUILD_TYPE=Release
 CMAKE_ARGS += -DCMAKE_C_COMPILER=$(TC_PATH)$(TC_PREFIX)gcc
 CMAKE_ARGS += -DCMAKE_CXX_COMPILER=$(TC_PATH)$(TC_PREFIX)g++
-CMAKE_ARGS += -DCMAKE_EXE_LINKER_FLAGS=-L$(INSTALL_DIR)$(INSTALL_PREFIX)/lib -Wl,--rpath-link,$(INSTALL_DIR)$(INSTALL_PREFIX)/lib -Wl,--rpath,$(INSTALL_PREFIX)/lib
 CMAKE_ARGS += -DCMAKE_FIND_ROOT_PATH=$(INSTALL_DIR)$(INSTALL_PREFIX)
 CMAKE_ARGS += -DCMAKE_FIND_ROOT_PATH_MODE_PROGRAM=NEVER
-CMAKE_ARGS += -DCMAKE_FIND_ROOT_PATH_MODE_LIBRARY=ONLY
-CMAKE_ARGS += -DCMAKE_FIND_ROOT_PATH_MODE_INCLUDE=ONLY
```

Comment:
Isn't there a risk other packages built with cmake support may no longer work? I would give it a check...

Reply:
Good catch.  My bets are that it doesn't but i will run a quick check to confirm all other packages migrated to cmake helper are ok based on PR https://github.com/SynoCommunity/spksrc/pull/3998

Reply:
@ymartin59 all non-ffmpeg related builds passed, all good.